### DISCUSSION TITLE
searchCatalogItems returns duplicate items

### DISCUSSION THREAD

**Discussion by LoseTheQuit** [amzn/selling-partner-api-models#4111] [General]
searchCatalogItems returns duplicate items

After using the nextToken, for about 5 times, the api starts to send duplicate results. After about the 20th page, the api returns no new results. Is this expected? If not, what could I be doing wrong?

**Comment by chapmanjw** [comment]
@LoseTheQuit Can you open a developer support case with specific request ids and timestamps of the requests for us to investigate in more detail? https://developer.amazonservices.com/support

Thanks
